src/globals/mixins.scss

```diff
@@ -8,6 +8,10 @@
   line-height: 24px;
 }
 
+@mixin philco-button-and-subhead-font() {
```

Comment:
Thought about best way to do this for a bit. I think it's quite good to have the option of a body text mixin that sets font size and this one which is agnostic to it. Didn't really seem any clearer to nest mixins when they only have 1 line in common.